cypress/integration/channel/header_spec.js

```diff
@@ -70,4 +70,36 @@ describe('Header', () => {
 
         cy.apiSaveMessageDisplayPreference();
     });
+    it('MM-13483 - Cleared search term should not reappear as RHS is opened and closed', () => {
+        // # Place the focus on the search box and search for something
+        cy.get('#searchFormContainer').click();
+        cy.get('#searchBox').type('London{enter}');
+
+        // # Clear the search text
+        cy.get('#searchBox').clear();
+
+        // # Verify the Search side bar opens up
+        cy.get('#sidebar-right').should('be.visible');
+        cy.get('#sidebar-right').should('contain', 'Search Results');
```

Comment:
Chain this additional validation with `and` off of the above one.
```suggestion
        cy.get('#sidebar-right').should('be.visible').and('contain', 'Search Results');
```

Reply:
done